api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -0,0 +1,120 @@
+package v1alpha1
```

Comment:
we had this conversation somehow already but as expected got lost between mails, chats, reviews. We need to have a (short) document in the docs dir of the repo which outlines how the conversion looks like and what to do when we need a new version bump.

Reply:
Until now we had a  "do nothing" as the API conversion strategy. Now we are doing the conversion ourselves. I do agree we need some doc about it, but other than that we are just following k8s conversion Webhook, nothing more.

Reply:
Which is perfectly fine and expected, but we need to minimize the "what I need to do now?" phase we can have like 6 months in the future when we will could do a new API bump.
We don't need to write a lot, even a basic README with few lines like "we don't do anything special. Just start with $OPERATOR_SDK_DOCUMENTATION_LINK and follow the steps" can be perfectly fine, but has to be in our repo and not lost in long threads/reviews.

Reply:
Will do, sure